test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -123,6 +123,78 @@ public void DotnetRestore_WithAuthorSignedPackage_Succeeds()
             }
         }
 
+        [Fact]
+        public async Task DotnetRestore_UpdateLastAccessTime()
+        {
+            using (var pathContext = _msbuildFixture.CreateSimpleTestPathContext())
+            {
+                var packageX = new SimpleTestPackageContext()
+                {
+                    Id = "TestPackage",
+                    Version = "1.0.0"
+                };
+
+                await SimpleTestPackageUtility.CreateFolderFeedV3Async(
+                    pathContext.PackageSource,
+                    PackageSaveMode.Defaultv3,
+                    packageX);
+
+                var projectName = "ClassLibrary1";
+                var workingDirectory = Path.Combine(pathContext.SolutionRoot, projectName);
+                var projectFile = Path.Combine(workingDirectory, $"{projectName}.csproj");
+
+                _msbuildFixture.CreateDotnetNewProject(pathContext.SolutionRoot, projectName, "classlib -f netstandard2.0");
+
+                using (var stream = File.Open(projectFile, FileMode.Open, FileAccess.ReadWrite))
+                {
+                    var xml = XDocument.Load(stream);
+
+                    var attributes = new Dictionary<string, string>() { { "Version", packageX.Version } };
+
+                    ProjectFileUtils.AddItem(
+                        xml,
+                        "PackageReference",
+                        packageX.Id,
+                        string.Empty,
+                        new Dictionary<string, string>(),
+                        attributes);
+
+                    ProjectFileUtils.WriteXmlToFile(xml, stream);
+                }
+
+                // set nuget.config properties
+                var doc = new XDocument();
+                var configuration = new XElement(XName.Get("configuration"));
+                doc.Add(configuration);
+
+                var config = new XElement(XName.Get("config"));
+                configuration.Add(config);
+
+                var forceUpdatePackageLastAccessTime = new XElement(XName.Get("add"));
+                forceUpdatePackageLastAccessTime.Add(new XAttribute(XName.Get("key"), "forceUpdatePackageLastAccessTime"));
+                forceUpdatePackageLastAccessTime.Add(new XAttribute(XName.Get("value"), "true"));
+                config.Add(forceUpdatePackageLastAccessTime);
+
+                File.WriteAllText(Path.Combine(workingDirectory, "NuGet.Config"), doc.ToString());
```

Comment:
Does this overwrite the config created by SimpleTestPathContext?
If it does, then the test will leak packages into the user's globla packages folder.

Reply:
Just checked and there is no nuget.config file at this location `NuGet.Client\.test\work\f4003c4c\c8077edf\solution\ClassLibrary1\NuGet.Config`, the file is created not overwritten.

There is only `Class1.cs` and `ClassLibrary1.csproj`.

Reply:
Ah, ok. 

There should be a config automatically created in NuGet.Client\.test\work\f4003c4c\c8077edf\NuGet.Config. 

Basically it's supposed to isolate the test from the machine configurations.

Reply:
For reference, what threw me off is that SimpleTestPathContext has a working directory which is different from the one that's named working directory here: https://github.com/NuGet/NuGet.Client/blob/db1acf828845c4f6177d3acae0abab1ac6b9a1c5/test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestPathContext.cs#L17.